rotkehlchen/tests/external_apis/test_defillama.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+from rotkehlchen.constants.assets import A_DAI, A_ETH, A_EUR, A_USD
+from rotkehlchen.fval import FVal
+from rotkehlchen.types import Price
+
+aave_mocked_historical_prices = {
```

Comment:
this var name makes no sense. Copy pasta?

Reply:
Yes